src/components/RichTextEditor/__tests__/ListPlugin.cy.tsx

```diff
@@ -22,15 +29,41 @@ const RichTextEditorWithListPluginWithSoftBreak = ({ value }: { value: string })
     return <RichTextEditor border={false} plugins={plugins} value={value} />;
 };
 
-const RichTextEditorWithUnorderedListStyles = () => <RichTextEditor value={JSON.stringify([unorderedListValue])} />;
-const RichTextEditorWithOrderedListStyles = () => <RichTextEditor value={JSON.stringify([orderedListValue])} />;
+const plugins = new PluginComposer();
+plugins
+    .setPlugin(new SoftBreakPlugin())
+    .setPlugin(
+        new TextStylePlugin({
+            textStyles: [
+                new Heading1Plugin(),
+                new ParagraphPlugin(),
+                new Custom1Plugin(),
+                new Custom2Plugin(),
+                new Custom3Plugin(),
+            ],
+        }),
+    )
+    .setPlugin([new UnorderedListPlugin(), new OrderedListPlugin()]);
+
+const RichTextEditorWithUnorderedListStyles = () => (
+    <RichTextEditor value={JSON.stringify([unorderedListValue])} plugins={plugins} />
+);
+const RichTextEditorWithOrderedListStyles = () => (
+    <RichTextEditor value={JSON.stringify([orderedListValue])} plugins={plugins} />
+);
+
+import './fixtures/theme.css';
```

Comment:
Could this import be at the same place which is on top, if this exist in any other file, please do that to.

Reply:
Already moved to the top 👍